runtime/pom.xml

```diff
@@ -145,6 +145,7 @@
             SpecV1AddressTest.test233, SpecV1AddressTest.test234, SpecV1AddressTest.test236,
             SpecV1AddressTest.test252, SpecV1AddressTest.test253, SpecV1AddressTest.test255
           </excludedTests>
+          <excludedValidationTests>binary,binary-leb128,custom</excludedValidationTests>
```

Comment:
Since we are excluding at the file level is it possible to have the filename listed here?
e.g. `binary.wast, custom.wast,`

Reply:
Done.